migrations/2_deploy_contracts.js

```diff
@@ -1,5 +1,5 @@
-var ECommerce = artifacts.require("./ECommerce.sol");
+var MyContract = artifacts.require("./MyContract.sol");
```

Comment:
Why change the name?

Reply:
The directions have it stated that way, otherwise it should just be kept like that and the instruction to change the name should be removed